Parsers/ASimUserManagement/Parsers/vimUserManagementMicrosoftWindowsEvent.yaml

```diff
@@ -0,0 +1,242 @@
+Parser:
+  Title: User Management ASIM parser for Microsoft Windows Event logs
+  Version: '0.2.0'
+  LastUpdated: May 29, 2024
+Product:
+  Name: Microsoft
+Normalization:
+  Schema: UserManagement
+  Version: '0.1.1'
+References:
+- Title: ASIM User Management Schema
+  Link: https://aka.ms/ASimUserManagementDoc
+- Title: ASIM
+  Link: https://aka.ms/AboutASIM
+- Title: Audit User Account Management
+  Link: https://learn.microsoft.com/en-us/windows/security/threat-protection/auditing/audit-user-account-management
+Description: |
+  This ASIM parser supports normalizing Microsoft Security Event logs ingested in 'WindowsEvent' table delivered using AMA to the ASIM UserManagement normalized schema.
```

Comment:
please keep same description as ASIM parser:
This ASIM parser supports normalizing Microsoft Security Event logs ingested in 'WindowsEvent' table to the ASIM User Management normalized schema.